src/components/modals/WalletListModal.js

```diff
@@ -100,29 +120,42 @@ export function WalletListModal(props: Props) {
       showError(s.strings.network_alert_title)
     } else bridge.resolve({ walletId, currencyCode })
   })
-  const handleWithdrawToBank = useHandler(() => bridge.resolve({ isWithdrawToBank: true }))
   const handleSearchClear = useHandler(() => {
     setSearchText('')
     setSearching(false)
   })
   const handleSearchUnfocus = useHandler(() => setSearching(searchText.length > 0))
   const handleSearchFocus = useHandler(() => setSearching(true))
 
+  // Pull up the signup workflow on the calling scene if the user does not yet have a linked bank account
+  const handleShowBankPlugin = useHandler(() => bridge.resolve({ isBankSignupRequest: true }))
+
+  // #endregion Handlers
+
+  // #region Dependent Constants
+
   // Prevent plugins that are "watch only" from being used unless it's explicitly allowed
   const walletListExcludeAssets = useMemo(() => {
     const result = excludeAssets ?? legacyExcludeAssets
     return allowKeysOnlyMode ? result : KeysOnlyModeTokenIds.concat(result ?? [])
   }, [allowKeysOnlyMode, excludeAssets, legacyExcludeAssets])
 
+  // #region Bank Button/Row Display
+  const bankSignupButton = <MainButton label={s.strings.deposit_to_bank} type="secondary" onPress={handleShowBankPlugin} marginRem={[0.5, 0.75, 1.5, 0.75]} />
+  const bankSection = showWithdrawToBank ? (
+    <>
+      {linkedBankAccounts == null || Object.keys(linkedBankAccounts).length === 0 ? bankSignupButton : null /* TODO: Show payment methods, if available */}
+      <EdgeText>{s.strings.deposit_to_edge}</EdgeText>
+    </>
+  ) : null
```

Comment:
Eh, I don't see much benefit to not just inlining this component. It now looks like this is _state_ rather than _view_ code. I'd be okay with a "render" function (though keep in mind, I wasn't really into render functions either, but followed the convention).

Reply:
I think it really helps with self documentation to keep bigger blocks separate. Keeps related code that helps build these component groups together in one place, too. 

Besides convention and connotations of "render*" , is there any functional benefit to turning this into a function vs just having a constant?